pytest/tests/sanity/staking2.py

```diff
@@ -95,6 +95,7 @@ def doit(seq=[]):
                           ["block_producer_kickout_threshold", 40],
                           ["chunk_producer_kickout_threshold", 40]], {
                               0: {
+                                  "tracked_shards": [0],
```

Comment:
This doesn't make the node track shard 0, it makes it track all shards. It's terrible, I know.

Reply:
Oops, I was already living in `ProtocolFeature::SingleShardTracking` world.
But it doesn't make a big difference? This test has only one shard anyway and its purpose is to check `stake` txns correctness, not shards tracking.